content/kubernetes-backing-services/external-dns/external-dns.md

```diff
@@ -83,3 +83,23 @@ Add to your [Kubernetes Backing Services](/kubernetes-backing-services) Helmfile
 {{% include-code-block  title="helmfile.yaml" file="kubernetes-backing-services/external-dns/examples/external-dns-helmfile.yaml" language="yaml" %}}
 
 Then follow the instructions for running [`helmfile sync`]({{< relref "tools/helmfile.md" >}}).
+
+## Usage
+
+To leverage `external-dns`, you will need to add an annotations (e.g. `kubernetes.io/tls-acme: "true"`) to the `Ingress` resource.
+
+With these in place, then `kube-lego` will handle all e2e TLS certificate issueing and save the certificate from Let's Encrypt to a secret specificied by the `tls` config.
```

Comment:
With these in place, `kube-lego` will handle issuing of TLS certificates from Let's Encrypt and saving them to a Kubernetes secret specified by the `tls` config parameter